02-running-grakn/01-install-and-run.md

```diff
@@ -27,16 +27,16 @@ Update the package cache:
 
 Install Grakn Server and Grakn Console:
 ```
-# yum install grakn-core-server grakn-core-console
+# yum install grakn-core-all
 ```
 
 #### Using Debian
 
 As a superuser, add the repo:
 ```
 # apt install software-properties-common
-# add-apt-repository 'deb [ arch=all ] https://repo.grakn.ai/repository/apt/ trusty main'
 # apt-key adv --keyserver keyserver.ubuntu.com --recv 8F3DA4B5E9AEF44C
+# add-apt-repository 'deb [ arch=all ] https://repo.grakn.ai/repository/apt/ trusty main'
```

Comment:
I think we should remove all '#  `and` $ ` prefix from the terminal code, @max @soroush It makes it error-prone for those who copy-paste from the docs.

Reply:
@haikalpribadi how do we then signify that it needs to be ran as root?

Reply:
provide `sudo` as part of the command? @vmax

Reply:
I didn't even know that `$` indicates root!

Reply:
@haikalpribadi actually it's `#` that indicated root 🙂